sdk/cognitivelanguage/azure-ai-language-conversations/tests/async/test_conversation_app_async.py

```diff
@@ -14,76 +14,56 @@
 
 from azure.ai.language.conversations.aio import ConversationAnalysisClient
 from azure.ai.language.conversations.models import (
-    ConversationAnalysisOptions,
-    AnalyzeConversationResult,
+    CustomConversationalTaskResult,
     ConversationPrediction
 )
 
 
 class ConversationAppAsyncTests(AsyncConversationTest):
 
     @GlobalConversationAccountPreparer()
-    async def test_conversation_app(self, conv_account, conv_key, conv_project):
+    async def test_conversation_app(self, endpoint, key, conv_project_name, conv_deployment_name):
 
-        # prepare data
-        query = "One california maki please."
-        input = ConversationAnalysisOptions(
-            query=query,
-        )
-
-        # analyze quey
-        client = ConversationAnalysisClient(conv_account, AzureKeyCredential(conv_key))
-        async with client:
-            result = await client.analyze_conversations(
-                input,
-                project_name=conv_project,
-                deployment_name='production'
-            )
-        
-        # assert
-        assert isinstance(result, AnalyzeConversationResult)
-        assert result.query == query
-        assert isinstance(result.prediction, ConversationPrediction)
-        assert result.prediction.project_kind == 'conversation'
-        assert result.prediction.top_intent == 'Order'
-        assert len(result.prediction.entities) > 0
-        assert len(result.prediction.intents) > 0
-        assert result.prediction.intents[0].category == 'Order'
-        assert result.prediction.intents[0].confidence_score > 0
-        assert result.prediction.entities[0].category == 'OrderItem'
-        assert result.prediction.entities[0].text == 'california maki'
-        assert result.prediction.entities[0].confidence_score > 0
-
-    @GlobalConversationAccountPreparer()
-    async def test_conversation_app_with_dictparams(self, conv_account, conv_key, conv_project):
-        
-        # prepare data
-        query = "One california maki please."
-        params = {
-            "query": query,
-        }
-
-        # analyze quey
-        client = ConversationAnalysisClient(conv_account, AzureKeyCredential(conv_key))
+        # analyze query
+        client = ConversationAnalysisClient(endpoint, AzureKeyCredential(key))
         async with client:
-            result = await client.analyze_conversations(
-                params,
-                project_name=conv_project,
-                deployment_name='production'
+            query = "Send an email to Carol about the tomorrow's demo"
+            result = await client.analyze_conversation(
+                body={
+                    "kind": "CustomConversation",
```

Comment:
It seems weird that I'd always need to set this to the same value for every request... I'm wondering if we can default this unless it's expect to expand to more options?

Reply:
do you have an example of how to set default value using a directive in autorest swagger?
i kept looing for it but didn't find something useful in autorest docs/issues

Reply:
Looks like if we used the object rather than the dictionary input we would get this added to the payload for free. So maybe we could update a couple of the samples to demonstrate using the objects alongside the dictionary inputs?